.travis.yml

```diff
@@ -19,21 +17,34 @@ addons:
       - python-coverage
       - python-werkzeug
       - python-psycopg2
-      - postgis
-      - postgresql-9.4-postgis-2.1
+      - libgdal1-dev
```

Comment:
This package doesn't get installed because it conflicts with postgresql-9.4-postgis-2.2 dependencies